src/amber/cli/templates/app/.travis.yml

```diff
@@ -1 +1,6 @@
 language: crystal
+
+script:
+  - bin/ameba
```

Comment:
IMHO they should be put in reverse order.

Reply:
Do you mean run `crystal spec` first, then `crystal tool format --check`, then `bin/ameba`?

Reply:
Yes, that was my point. Because I reckon that the spec failures are more important then stylistic wrongdoings and you'd like to know about 'em as soon as possible.

Reply:
Interesting perspective.  Didn't think much about it to be honest.  Anyone else have opinions on this?  If not, I will reverse the order.

Reply:
Agree with @Sija, having `spec failures` is more important than coding conventions. Having any build `CI` notification before `ameba` run is better having them after (in term of response in case of bug)